otter/json_schema/group_schemas.py

```diff
@@ -151,7 +151,7 @@
             "description": (
                 "What type of a load balancer is in use"),
             "required": False,
-            "oneOf": ["CloudLoadBalancer"]
+            "pattern": "^CloudLoadBalancer$"
```

Comment:
It didn't work :|  Seems like the values themselves have to be schemas?  http://json-schema.org/latest/json-schema-validation.html#anchor88